### ISSUE TITLE
Support for libphp7.1 (external)

### ISSUE THREAD

**Issue by hackel** [cscott/node-php-embed#3]
This looks like an excellent project!  It's just what I want in my JS E2E tests to access my PHP backend functionality to setup test data, truncate tables, etc.  Is there any chance of it being updated to support libphp7.1?

Ideally it would use an externally installed library on the system by default, or when compiling from source it would automatically detect it and avoid trying to compile from scratch.  I need the embedded library to have access to all of the php modules installed on my system, such as mongodb.

**Comment by Maxim-Mazurok** [comment]
I don't think that you need to change library for that. Mongo is embedded using shared object, AFAIK.
